dev/public/index.html

```diff
@@ -15,8 +15,10 @@
     <nav class="bg-gray-800 pt-2 md:pt-1 pb-1 px-1 mt-0 h-auto fixed w-full z-20 top-0">
       <div class="flex flex-wrap items-center">
         <div class="flex flex-shrink md:w-1/3 justify-center md:justify-start text-white">
-          <img src="static/img/logo_64x64_white.png" alt="logo" class="logo h-11 m-2 ml-3">
-          <span class="h-11 m-2 py-1 text-3xl">BPMN Visualization</span>
+          <a class="home flex items-center" href="../../examples/index.html">
```

Comment:
⚠️  The `home` class is not available here.
We have the following css in the example repository:
```css
.navbar a.home:link {
    display: flex;
    align-items: center;
    text-decoration: none;
}
```

Reply:
@csouchet the `home` class still doesn't exist here. I am not sure we need to define `text-decoration: none` here.

Reply:
⚠️ the href is wrong when we use the demo in the preview or run it locally.

Reply:
After comparing with and without this class, i don't find any differences. so, I'll remove it.

Reply:
I don't know how to specify different href according to the environment.
We don't have home locally or in preview.